Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -977,15 +977,12 @@ public static void DestroyPlayerForConnection(NetworkConnection conn)
             // so we need null checks for both of them.
             // => destroy what we can destroy.
 
-            if (conn.clientOwnedObjects != null)
+            HashSet<uint> tmp = new HashSet<uint>(conn.clientOwnedObjects);
```

Comment:
@vis2k  copying the set seems unnecessary as @rodolphito points out.

Reply:
yes, I asked him to remove it in a separate pr